### ISSUE TITLE
import of "github.com/golang/protobuf/ptypes/any" No dependencies were provided.

### ISSUE THREAD

**Issue by USB-05** [bazelbuild/rules_go#3042]
### What version of rules_go are you using?

rules_go-v0.29.0.zip

### What version of gazelle are you using?

bazel-gazelle-v0.22.3.tar.gz

### What version of Bazel are you using?

bazel 4.2.2

### Does this issue reproduce with the latest releases of all the above?

Yes

### What operating system and processor architecture are you using?

Darwin UserName 21.2.0 Darwin Kernel Version 21.2.0: Sun Nov 28 20:28:54 PST 2021; root:xnu-8019.61.5~1/RELEASE_X86_64 x86_64

### What did you do?

Just Updated my WORKSPACE with above versions

### What did you expect to see?

A Successful Build

### What did you see instead?

`(15:56:40) ERROR: /private/var/tmp/_bazel_udhamsingh/c318b2e3ffcdc99819999255aa864929/external/go_googleapis/google/rpc/BUILD.bazel:38:17: GoCompilePkg external/go_googleapis/google/rpc/status_go_proto.a failed: (Exit 1): builder failed: error executing command bazel-out/host/bin/external/go_sdk/builder compilepkg -sdk external/go_sdk -installsuffix darwin_amd64 -src ... (remaining 49 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
compilepkg: missing strict dependencies:
        /private/var/tmp/_bazel_udhamsingh/c318b2e3ffcdc99819999255aa864929/sandbox/darwin-sandbox/3296/execroot/harness_monorepo/bazel-out/darwin-fastbuild/bin/external/go_googleapis/google/rpc/status_go_proto_/google.golang.org/genproto/googleapis/rpc/status/status.pb.go: import of "github.com/golang/protobuf/ptypes/any"
No dependencies were provided.
Check that imports in Go sources match importpath attributes in deps.`

**Comment by robfig** [comment]
This seems likely to be a dupe of https://github.com/bazelbuild/rules_go/issues/2927. Any assistance with the analysis and a PR would be much appreciated! Thanks for the report

**Comment by USB-05** [comment]
Yes i know but in #2927 . The author is claiming that the patch is working. However i tried the same patch and it is not working for me :(

**Comment by debuggerpk** [comment]
can confirm with @USB-05 .. not working

**Comment by m9rco** [comment]
I came across the same problem
